### DISCUSSION TITLE
Build without use strict

### DISCUSSION THREAD

**Discussion by isolin** [parcel-bundler/parcel#7375] [Q&A]
I use [LAME.js](https://github.com/zhuker/lamejs) in my project to create render music to audio files. It's a pure web app without backend.

With parcel v1 the builds went fine, but when I upgraded to v2 I started getting an error in all browsers, e.g. in FF `Uncaught ReferenceError: assignment to undeclared variable Lame`. The reason is that the LAME package is 4 years old and it does not support the `strict `mode. It just makes assignments to undeclared variables like `Lame = require('./Lame.js');`

How can I deal with that? Is there a way to drop the `use strict` for that package or worst case in general? I tried to set it in `.tsconfig` but it had no effect on the mentioned error.

**Comment by cancerberoSgx** [comment]
similarly here when importing a webassembly library which contain octal escapes. Why are you adding a "use strict" statement in my code to make it fail like this?

`Uncaught SyntaxError: Octal escape sequences are not allowed in strict mode. (at index.2d3ace14.js:63809:67)`
